### PR TITLE
1273 printevenarguments 1

### PR DESCRIPTION
**Pull request by eslopfer**
Add tests and solution for printevenarguments subject.

### CODE REVIEW

**Review by Hamzaelkhatri — APPROVED** [review]

File: tests/printevenarguments_test/main.go [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package main
+
+import "github.com/01-edu/go-tests/lib/challenge"
+
+func main() {
+	table := [][]string{
```
Comment (MSilva95) [inline tests/printevenarguments_test/main.go:6]: Add a test to cover this case
```
$ go run . a | cat -e
```

File: solutions/printevenarguments/main.go [hunk 2]
```diff
@@ -0,0 +1,18 @@
+package main
```
Comment (MSilva95) [inline solutions/printevenarguments/main.go:1]: Change the solution if necessary to cover the case i said in the readme

**Review by MSilva95 — CHANGES_REQUESTED** [review]

File: tests/printevenarguments_test/main.go (hunk 1 above)
Comment (eslopfer) [reply tests/printevenarguments_test/main.go:6]: Thanks I'll do that.

File: solutions/printevenarguments/main.go (hunk 2 above)
Comment (eslopfer) [reply solutions/printevenarguments/main.go:1]: Done. Thank you.

**Review by MSilva95 — APPROVED** [review]

**Pull request merged**